components/automate-gateway/api/nodes/manager/manager.proto

```diff
@@ -56,6 +89,13 @@ service NodeManagerService {
 			action: "infra:nodeManagers:update"
 		};
 	};
+
+	/*
+	Delete a nodemanager
+
+	Delete a nodemanager given an id. Note this only deletes the manager itself. Any nodes
+	associated with the manager will be abandoned.	
```

Comment:
Would "orphaned" be better than "abandoned" ? Not familiar enough with the ramifications.

Reply:
could we say what abandoned means instead of using a different word?